docs/ROADMAP.md

```diff
@@ -11,7 +11,7 @@ Below, you can see the key milestones we anticipate.
 | ----------------------------------- |:-:| ------------: |
 | First Early Access Preview (EAP)    | ✅ |     July 2024 |
 | Second Early Access Preview (EAP 2) | ✅ | November 2024 |
-| Third Early Access Preview (EAP 3)  | 🏗️ |    April 2025 |
+| Third Early Access Preview (EAP 3)  | ✅ |    April 2025 |
 | Incubating Software Types and DCL   | ⏳ |          2025 |
```

Comment:
🤔 Composability/Extensibility is a question I get from just about everyone I talk to about D-G.  Is it worth an "in-progress" entry here?